container/gitpod/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM opensuse/leap:15.3
```

Comment:
why not inherit from the devel:openQA:ci container instead of copy-pasting content?

Reply:
that… was far too obvious…
sorry, fixed now.